article/background.tex

```diff
@@ -80,13 +81,31 @@ \subsection{Software Dependencies}
 
 \subsection{Containers}
 
-Operating system virtualization is a process whereby an operating system can be installed inside another operating system, without being subject to the environment constraints of its parent, and without potentially polluting the parent environment.
-Given the complexity of scientific computing environments, such virtualization is attractive on multi-user systems, on systems lacking adequate package management capabilities, or in instances where the tasks to be executed are fragile and may require bespoke constraints.
+Operating system virtualization is a process whereby an operating system can be emulated inside another running system, the "host", and thus a "guest" environment be shared with any software and dependencies already installed.
+Virtual machines (VMs) are attractive solutions to enable reproducibility for several reasons:
 
-A prominent instantiation of virtualization is container technology, which focuses on portability of operating system images (the eponymous “containers”) across parent operating systems contingent only on the presence of a container running service.
-Their relevance to open science consists in providing end-users with an accessible environment, which can be ascertained to provide the requirements of a certain top-level workflow, and which does not interfere with their parent environment.
+1. VMs skip the most difficult step: Installation and coordinating dependencies is often challenging, time consuming, and requires domain knowledge reproducers and reviewers are likely to lack.
+1. Guests are self-contained and isolated from the host, which eliminates the posibility of polluting the host environment.
+1. Admins are able to safely allow mostly unrestricted usage from semi-trusted users.
+1. Package managers are imperfect, and none can be relied upon to continue to offer all legacy releases, but VMs preserve a working copy.
 
-Two of the most prominent container standards are offered by the Open Container Initiative (OCI) and Singularity.
-The former is used by applications such as Docker and Podman, and the latter by the eponymous Singularity application, which differentiates itself from OCI by not requiring root access, and thus being arguably better suited for scientific computing in high-performance computing environments.
+System virtualization offers a way to portably freeze and preserve environments, but are limited due to the size of the "full disk images".
+Additionally VM's must be be "booted" which can be costly if many instances are needed.
+Modern advances in container technology have allowed similar benefits but strip redundancy by making limited use of the host machine, specifically the hypervisor.
```

Comment:
Do we need to go into this much detail, if so shouldn't we explain what a hypervisor is?
I think if we explain containers, we should do so only with respect to what is relevant for our usage/workflow construction.
Similarly to e.g. the fMRI data analysis explanation, there's a lot more to it, but really only what's common to the general and the specific case here should be expanded on.

Reply:
I was going for "vague thing that's enough to understand what we're talking about, or if you are interested Google this word".